plugins/optimization-detective/docs/introduction.md

```diff
@@ -186,7 +186,7 @@ add_filter( 'query_vars', function ( $query_vars ) {
 
 Then in your template code you may continue to use `isset( $_GET['thank_you'] )` but you might as well use `get_query_var( 'thank_you' )` instead.
 
-When an `od_url_metrics` post has not been updated in a month then it is garbage-collected, since it is likely the original URL has gone away.
+When an `od_url_metrics` post has not been updated within the default value i.e., a month or the filtered value via `od_url_metric_garbage_collection_ttl` then it is garbage-collected, since it is likely the original URL has gone away.
```

Comment:
This filter should be added to `hooks.md` and include docs like this sample code:

```php
add_filter( 'od_url_metric_garbage_collection_ttl', function () {
	return 6 * MONTH_IN_SECONDS;
} );
```

As well as:

```php
add_filter( 'od_url_metric_garbage_collection_ttl', '__return_zero' );
```